src/parser/serializer.ts

```diff
@@ -583,13 +689,19 @@ function serializeBinary(buffer, key, value, index, isArray?: boolean) {
   }
 
   // Write the data to the object
-  data.copy(buffer, index, 0, value.position);
+  buffer.set(data, index);
```

Comment:
This was a bug in the browser bundles, data can be a Uint8Array, I tried going back to the old bundles to figure out how this worked but I couldn't get it to work.
Anyway `set` is the Uint8Array equivalent of buffer's copy and is available on both.

Reply:
This might be fixed with the proposal to use `Buffer` everywhere